program/rust/src/tests/test_publish.rs

```diff
@@ -73,10 +73,9 @@ async fn test_publish() {
         assert_eq!(price_data.comp_[0].latest_.price_, 150);
         assert_eq!(price_data.comp_[0].latest_.conf_, 7);
         assert_eq!(price_data.comp_[0].latest_.status_, PC_STATUS_TRADING);
-
-        assert_eq!(price_data.comp_[0].agg_.price_, 0);
-        assert_eq!(price_data.comp_[0].agg_.conf_, 0);
-        assert_eq!(price_data.comp_[0].agg_.status_, PC_STATUS_UNKNOWN);
+        assert_eq!(price_data.comp_[0].agg_.price_, 150);
```

Comment:
Note: you can update test_publisher_set.rs to remove the `// Final price update to trigger aggregation` part since we don't need to trigger the aggregation anymore